pkgs/development/python-modules/blinkpy/default.nix

```diff
@@ -26,6 +27,11 @@ buildPythonPackage rec {
     hash = "sha256-r8kf5L6bvtivqd9dSi8om1wIi8IHmipKFckNMPT515I=";
   };
 
+  nativeBuildInputs = [
+    setuptools
+    wheel
```

Comment:
```suggestion
```

That should not be required. https://github.com/pypa/setuptools/commit/f7d30a9529378cf69054b5176249e5457aaf640a